frontend/app/pages/index.vue

```diff
@@ -3,9 +3,9 @@ Copyright (c) 2025 The Linux Foundation and each contributor.
 SPDX-License-Identifier: MIT
 -->
 <template>
-  <lfx-collection-list-view />
+  <lfx-explore />
```

Comment:
This wont be enough. You have to change values in LfxRoutes aswell. Since currently COLLECTIONS = 'index' you have to make explore have index value and collections have collections value